common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java

```diff
@@ -231,7 +231,7 @@ public static String getStackTraceAsString(@NonNull final Exception exception) {
         return pw.toString();
     }
 
-    public static String fromByteArray(@NonNull final byte[] bytes) {
```

Comment:
So Nullable?

Reply:
what happens if this is NULL?

Reply:
a warning is shown if we put NonNull here, so I removed it.